### DISCUSSION TITLE
全球最大 3D 列印工程！中國計劃在青藏高原透過 AI 建造 3D 列印大壩

### DISCUSSION THREAD

**Discussion by blackline0911** [1101-GS4719-C/Python-Community-GS4719-C#641] [General]
https://technews.tw/2022/05/14/china-eyes-ai-powered-3d-printed-dam-on-tibetan-plateau/

據報導，中國正在利用 AI 人工智慧技術，將青藏高原大壩工程（羊曲水電站）變成一個完全不需工人協助建造的全球最大 3D 列印工程。預計該水電站將在 2024 年完成

多年來，科學家一直透過 AI 來研究水壩的運行效率，尤其是用於監控上最多，以定位出諸如堤坊及暴雨等會對水壩造成影響的風險因子。此外，AI 也有助於管理人員透過數據遠端追蹤水壩的維護與維修。

中國非常依賴水力發電，水力發電是僅次於燃媒發電的第二大能源來源，中國政府正面臨著 2060 年實現碳中和目標的龐大壓力，預計 2024 年完工的羊曲水電站，除了能為擁有 1 億人口的河南省輸送 50 億瓩時電力外，也能稍稍緩解上述壓力

**Comment by 11timyyy** [comment]
在列印的同時，也要控管好品質

**Comment by bloodtrail0511** [comment]
雖然方便但安全性不知道怎麼樣

**Comment by stan890425** [comment]
3D列印相較傳統的建造方法應該會更省材料，但安全性跟耐用性就無從得知了~

**Comment by sys511613** [comment]
這個真的很厲害，不過施工的時候可能會有很多突發狀況導致機器無法解決，所以人類的監控還是很重要的。但是透過無人挖土機、卡車、推土機、鋪路機及壓路機等還是可以節省大量人力資源，而且在比較簡單的項目機器或許會做的比人類更好。

**Comment by leon10456** [comment]
感謝樓主分享

**Comment by etin2468** [comment]
那要負責列印的機器也要超大ㄟ，感覺很壯觀

**Comment by Lynn12378** [comment]
沒想到列印也能做到這樣，就是耐久度不知道如何

**Comment by Hchejxnc** [comment]
如果之後使用都沒發生問題的話，覺得這會是3D列印技術的一大進步。

**Comment by Jimmy911218** [comment]
3D列印確實效率快但也要注意品質材行

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
3D列印的技術效率會比傳統更加快速 但要做出大霸 不知道要用多大的3D列印的機器

**Comment by Pietra1226** [comment]
原來3D列印已經可以應用到體積這麼大的東西上了，希望品質是好的，免得受苦的是民眾。

**Comment by ghost** [comment]
如果發展得當，將能改變整個建築體係，但如果品管不行的話，也將可能引發巨大的災難

**Comment by delia07152** [comment]
沒有想過3D列印能用在建造龐大的水壩，聽起來很像天方夜譚，若真的建完一定能造成極大的回響，不過後續是否真的安全，也會成為大家關注的焦點。

**Comment by bonny123865** [comment]
感覺如果真的打造起來會是一個很有幫助的建設欸!
但是如何抉擇材料的選用跟評估耐用度也是一個重點

**Comment by Kotori0501** [comment]
沒有想過3D列印技術可以運用在這種地方
但還是要考慮到材質等等是否能穩固
不然災害來了，倒了可就麻煩了

**Comment by manipulateed** [comment]
工程的效應確實能達到降低污染和發電充足，工程完成還要安全評估等等確保下游人民和財產的保障無疑

**Comment by ruthchen1020** [comment]
青藏高原上做大壩不合適吧。
首先大壩非常破壞環境，青藏高原生態脆弱應該不適合，再者青藏高原上的水直接影響地下的東南亞國家的河流資源，這是要底下的國家怎麼活?
中共這個想法真的很爛。

**Comment by StevenWanglolz** [comment]
3D列印的...水壩? 感覺安全上存在著一些疑慮~
